support-frontend/assets/helpers/subscriptionsForms/rules.ts

```diff
@@ -265,6 +265,13 @@ function applyDeliveryRules(fields: FormFields): Array<FormError<FormField>> {
 				'Please indicate whether the billing address is the same as the delivery address',
 			),
 		},
+		{
+			rule: zuoraCompatibleString(fields.deliveryInstructions),
+			error: formError(
+				'deliveryInstructions',
+				'Please use only letters, numbers and punctuation.',
+			),
+		},
```

Comment:
We weren’t testing the delivery instructions for emojis. I haven’t conclusively reviewed all fields on the frontend to make sure we’re doing this validation for them, though I have attempted to on the backend.